doc/Settings.md

```diff
@@ -263,3 +263,14 @@ You can enable the feature as shown below.
        "pinning": true
    },
 ```
+
+### windowsFeature
+
+This feature enables the ability to enable Windows Feature dependencies during installation.
+You can enable the feature as shown below.
+
+```json
+   "experimentalFeatures": {
+       "windowsFeature": true
```

Comment:
Features are a code construct whose state is driven by settings.  Settings can theoretically be arbitrarily complex, but I don't think we have any that are not simple types currently.  The settings code could even handle both bool and object values for `dependencies` if we were inclined.

Reply:
> The settings code could even handle both bool and object values for dependencies if we were inclined.

So you're saying it would be possible to have it where
```json
"experimentalFeatures": {
  "dependencies": true
}
```
Would either enable all dependencies, or keep the current behavior of only enabling package dependencies

OR, users could specify individual dependencies features?
```json
"experimentalFeatures": {
  "dependencies": {
    "packageDependencies" : true
    "windowsFeatures": false
  }
}
```

Reply:
Yes, it would be possible to write the code the enable this.  I'm not signing @ryfu-msft up for that though 😄